### PR TITLE
Add warning

### PR DESCRIPTION
**Pull request by hwwhww**
### What was wrong?

We should warn the users.

#### Cute Animal Picture

🕸️ 🕷️

### CODE REVIEW

**Review by CarlBeek — COMMENTED** [review]
I like this idea, but why specifically is the warning only in the BLS sections? Has the rest of py_ecc been audited?

Additionally, we should update line 3 to include a reference to the bls12-381 curve in addition to secp256k1 and alt_bn128.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 py_ecc
 ==========================
-Elliptic curve crypto in python including secp256k1 and alt_bn128
+Elliptic curve crypto in python including secp256k1, alt_bn128, and bls12_381.
+
+**Warning: It contains some experimental codes and has NOT been audited.**
```
Comment (pipermerriam) [inline README.md:5]:
```suggestion
> **Warning**: This library contains some experimental codes that have **NOT** been audited.
```

**Review by pipermerriam — APPROVED** [review]
Maybe good to be specific about what has/not been audited.  Has any of this been audited?

**Comment by hwwhww** [comment]
> Has any of this been audited?

Afaik, I think no.

**Pull request merged**